### DISCUSSION TITLE
Contributing New Components

### DISCUSSION THREAD

**Discussion by TrevorJoelHarris** [microsoft/fluentui#21089] [Q&A] [answered]
If my team was considering adding brand new fluent components, would it make sense to start building them here (https://github.com/microsoft/fluentui/tree/master/packages/react-components), or is it better to start building them in the React/React Northstar/Web Components packages? Does it matter, or is it up to us?

**Comment by khmakoto** [comment, marked as answer]
Hi @TrevorJoelHarris, any new components would ideally follow our [Component Convergence Guide](https://github.com/microsoft/fluentui/wiki/Component-Convergence-Guide). Feel free to take a look and let us know if you have any further questions (in which case we'd be more than happy to update our wiki 😄).
